frontend/packages/console-shared/src/hooks/__tests__/usePinnedResources.spec.ts

```diff
@@ -65,6 +74,7 @@ describe('usePinnedResources', () => {
   });
 
   it('returns some default pins if there are no other pins defined and the extension has default pins', async () => {
+    window.SERVER_FLAGS.perspectives = '[{ "id" : "dev", "visibility": {"state" : "Enabled" }}]';
```

Comment:
No customization for old tests. This also ensures that our code still works without it.

```suggestion
    window.SERVER_FLAGS.perspectives = '';
```